requirements.txt

```diff
@@ -1,19 +1,27 @@
 asgiref==3.7.2
 attrs==23.1.0
+bcrypt==4.1.2
+blinker==1.7.0
 boto3==1.34.0
 botocore==1.34.0
 cachetools==5.3.2
 certifi==2023.11.17
 cffi==1.16.0
 charset-normalizer==3.3.2
+click==8.1.7
 cryptography==41.0.7
 Django==5.0
+djangorestframework==3.14.0
 dnspython==2.4.2
+Flask==3.0.0
```

Comment:
just curious.

Why do we need to use Flask here?

Reply:
Oh sorry! Originally I was going to use flask for part of my user authentication, but I ended up not using it. I will remove!